### ISSUE TITLE
Allowing projects to release only derivatives but not raw data

### ISSUE THREAD

**Issue by francopestilli** [brain-life/warehouse#90]
Under specific situations, a user might be interested in sharing derivatives computed using brain-life apps, but NOT the raw data. This could be data from clinical populations or other sources that do not provide sufficient privileges to re-share the original data but do allow sharing of data derivatives.

**Comment by soichih** [comment]
We've been storing raw data and derivative data on separate projects. I think this makes sense because we want to treat the raw data project to be immutable (we probably don't want anyone to change it) with its own access policy, and derivative projects to have another set of access policy (probably only accessible by the project member). There will be more than 1 derivative projects that uses the same raw data projects, and derivative projects may actually use data from more than 1 raw data projects (maybe combining HCP 3T and 7T projects data?)

Basically, "project" can be used as more of a collection of datasets with distinctive access policy. 

In the clinical data case, I believe we could ask the researchers to host the raw data in one project with a very restrictive access policy (very few people can read/write), and create another project to host all derivative that they may make it public or with less strict access policy. 

Would such approach work for your use-case?

**Comment by soichih** [comment]
Closing this for now.
